addons/point_of_sale/models/product.py

```diff
@@ -3,10 +3,15 @@
 from odoo import api, fields, models, _
 from odoo.exceptions import UserError
 
+# fields that cannot be modified when there is a running pos session
+FORBIDDEN_PRODUCT_FIELDS = set([
+    'available_in_pos', 'pos_categ_id'
```

Comment:
`active`?

Reply:
This is arguable, In my tests it did not cause any problems to archive products.